runtime/doc/news.txt

```diff
@@ -167,6 +167,8 @@ The following changes to existing APIs or features add new behavior.
 
 • API calls now show more information about where an exception happened.
 
+• Strings in :intro are now friendlier to translators.
```

Comment:
news.txt is introduced during 0.9 release cycle, so this PR doesn't need a news.txt update. It's fine to just let the new.txt workflow fail.